Formula/qt.rb

```diff
@@ -77,6 +101,10 @@ def install
       -no-sql-psql
     ]
 
+    on_macos do
+      config_args << "-sysroot #{MacOS.sdk_path}"
```

Comment:
```suggestion
      config_args << "-sysroot" << MacOS.sdk_path.to_s
```
Push this later; we've already got a bunch of long timeout jobs running currently.